### PR TITLE
Jk/cumulus 3148 update tf module

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** 

This PR updates the `cumulus-rds-tf` module to utilize updated postgres 13 module defaults, and provides migration guidance from users who updated or were forcibly updated as part of the AWS database migration. 

Addresses [CUMULUS-3148](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3148)

This PR is a forward port of https://github.com/nasa/cumulus/pull/3234 and should be reviewed in tandem. 

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: example/rds-cluster-tf/variables.tf [hunk 1]
```diff
@@ -1,7 +1,3 @@
-variable "prefix" {
-  type = string
-}
```
Comment (npauzenga) [inline example/rds-cluster-tf/variables.tf:3]: Nice. I wonder how many other duplicates there are in our TF modules.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### MIGRATION notes
+
+From this release forward Core will be tested against postgres 11   Existing
```
Comment (npauzenga) [inline CHANGELOG.md:11]:
```suggestion
From this release forward Core will be tested against PostgreSQL 11   Existing
```
I know we're not strict about this kind of thing so feel free to ignore this kind of change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+release compatibility testing was done for release 11.1.8/14.0.0+.   Users
+should migrate their datastores to Aurora postgres11.13+ compatible data stores
```
Comment (npauzenga) [inline CHANGELOG.md:13]:
```suggestion
should migrate their datastores to Aurora PostgreSQL 11.13+ compatible data stores
```
Unless that's now what AWS calls it in the console.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+If you manually upgraded and the cluster is now on version 11.13, to continue using the `cumulus-rds-tf` module *once
+upgraded* update following module configuration values if set, or allow their
+defaults to be utilized:
+
+```terraform
+parameter_group_family = "aurora-postgresql11"
+engine_version = 11.13
+```
+
+When you apply this update, the original postgresql v10 parameter group will be
```
Comment (npauzenga) [inline CHANGELOG.md:31]:
```suggestion
When you apply this update, the original PostgreSQL v10 parameter group will be
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -42,6 +68,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- **CUMULUS-3148**:
+  - Updates cumulus-rds-tf to use defaults for postgres 11.13
```
Comment (npauzenga) [inline CHANGELOG.md:72]:
```suggestion
  - Updates cumulus-rds-tf to use defaults for PostgreSQL 11.13
```

**Review by npauzenga — APPROVED** [review]
This looks good @Jkovarik. I left a couple optional comments in the changelog and it looks like there's a conflict but otherwise it's good to go.

So the testing we've done is complete and we've already upgraded our unit test container https://github.com/nasa/cumulus/pull/3232/files. Is there anything else we need to do before release?

**Comment by Jkovarik** [comment]
@npauzenga  this should be releasable.

**Pull request merged**